eng/testing/tests.wasm.targets

```diff
@@ -79,4 +81,17 @@
       <WasmFilesToIncludeInFileSystem Include="@(WasmFilesToIncludeFromPublishDir -> '$(PublishDir)%(Identity)')" />
     </ItemGroup>
   </Target>
+
+  <!-- linker automatically picks up the .pdb files, but they are not added to the publish list.
+       Add them explicitly here, so they can be used with WasmAppBuilder -->
+  <Target Name="AddPdbFilesToPublishList" AfterTargets="ILLink">
```

Comment:
should we make this conditional on `DebuggerSupport == true`?

Reply:
I looked at the sdk targets and now I'm confused because if I'm reading this correctly pdbs should already be added to ResolvedFileToPublish by default: https://github.com/dotnet/sdk/blob/40f28f2de8dac1b2297a73b072dbc08f27a181a6/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L558-L564

Reply:
Maybe these pdb files are in different location after linking? I didn't find how the `_DebugSymbolsIntermediatePath` is set, so I am not sure.

Reply:
> I looked at the sdk targets and now I'm confused because if I'm reading this correctly pdbs should already be added to ResolvedFileToPublish by default: https://github.com/dotnet/sdk/blob/40f28f2de8dac1b2297a73b072dbc08f27a181a6/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L558-L564

That's copying the pdb for the main assembly only. I'm adding them for the files from other locations, like copy local ones (in this case from the runtime pack).

Reply:
Ah that makes sense, thanks.